### DISCUSSION TITLE
Connect through wifi dongle

### DISCUSSION THREAD

**Discussion by erdeidominik99** [wlcrs/huawei_solar#224] [Setting up the integration] [answered]
Hello!
I updated the inverter and the dongle to the latest firmware (V100R001C00SPC148, V100R001C00SPC130), but the option "Local O&M" is not visible in the settings->communication configuration->router settings of the app, and the port 6607 is not opened. What can be the problem?

**Comment by wlcrs** [comment]
Make sure to have the latest version of the FusionSolar app. I currently have version 6.23.00.161.

**Reply by erdeidominik99** [reply]
I updated the app, but nothing changed, I only have the wifi settings and the ip settings there.

**Comment by wlcrs** [comment, marked as answer]
My firmware version is V200R001C00SPC126. I guess it will not be available
on V200.... firmware and not on V100..... firmware. I'll need to update the
Wiki to reflect that.

**Reply by erdeidominik99** [reply]
V200 is for SUN2000L and v100 is for SUN200MA I think.

**Reply by wlcrs** [reply]
I've updated the Wiki: https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-the-inverter/_compare/05ce6569d5a9c44388c0a936e9ef7e4001c7604f...3c7b7133a036af17a5fd19b1d29c7c4e46752ba6

**Reply by erdeidominik99** [reply]
Okay, do you know which was the latest firmware for the dongle when this worked? Maybe I can doengrade the dongle.

**Comment by erdeidominik99** [comment]
Hello! I played with this, and turned out that I can read the registers with QmodMaster, but when I try to set up the integration It says "Reading from the inverter failed.".
